view/templates/admin/federation.tpl

```diff
@@ -36,15 +63,42 @@ var myDoughnutChart = new Chart(ctx).Doughnut(FedData, { animateRotate : false,
     </tr>
     <tr>
     <td colspan="3" class="federation-data">
-    <canvas id="{{$c[2]}}Chart" class="federation-network-graph"></canvas>
+    <canvas id="{{$c[2]}}Chart" class="federation-network-graph" width="240" height="240"></canvas>
     <script>
-    var {{$c[2]}}data = [
-    {{foreach $c[1] as $v}}
-	{ value: {{$v['total']}}, label: '{{$v['version']}}', color: "{{$c[3]}}", highlight: "#EE90A1",},
-    {{/foreach}}
-    ];
+	var {{$c[2]}}data = {
+		datasets: [{
+			data: [
+	{{foreach $c[1] as $v}}
+				{{$v['total']}},
+	{{/foreach}}
+			],
+			backgroundColor: [
+	{{foreach $c[1] as $v}}
+				'{{$c[3]}}',
+	{{/foreach}}
+			],
+			hoverBackgroundColor: [
+	{{foreach $c[1] as $v}}
+				'#EE90A1',
+	{{/foreach}}
+			]
+		}],
+		labels: [
+	{{foreach $c[1] as $v}}
+			'{{$v['version']}}',
+	{{/foreach}}
+		]
+	};
     var ctx = document.getElementById("{{$c[2]}}Chart").getContext("2d");
-    var my{{$c[2]}}DoughnutChart = new Chart(ctx).Doughnut({{$c[2]}}data, {animateRotate : false,});
+	var my{{$c[2]}}DoughnutChart = new Chart(ctx, {
```

Comment:
the indentation looks wrong in githubs diff. can you check?

Reply:
Indeed, I used tabs when it was spaces.